src/ocaml/parsing/pprintast.ml

```diff
@@ -1677,8 +1709,9 @@ let core_type = core_type reset_ctxt
 let pattern = pattern reset_ctxt
 let signature = signature reset_ctxt
 let structure = structure reset_ctxt
+let module_expr = module_expr reset_ctxt
 let case_list = case_list reset_ctxt
-let module_ = module_expr reset_ctxt
+let module_ = module_expr (* for merlin *)
```

Comment:
`module_ = module_expr`...
Maybe we can remove `module_` (added for merlin), now that `module_expr` is exported and used by the upstream typechecker (for printing diffs it seems)?